workflows/energyplus.py

```diff
@@ -154,7 +233,8 @@ def eplus_suffixes():
             "Map.txt",
         ]
 
-    def run_energyplus(self, isIP, run_directory, file_name, args, by_api):
+    @staticmethod
```

Comment:
An overall cleanup of this workflow file, not much functional change, mostly refactoring.